Dynatrace/dynatrace-operator
# Bump version of ubi9 base image

# Description

Bumps the version of the used `ubi-minimal` and `ubi-micro` images from 9.0.0 to 9.1.0.
Cherrypicked changes from the corresponding dependabot PRs.

## How can this be tested?

* Build the image from this branch making sure no build errors occur
* Deploy the operator
* Make sure the operator does not explode

## Checklist
  - N.A.
